### ISSUE TITLE
A11y_VSCode_SidebarBarOptions_AccessingTheExtensionbaroptions_ScreenReader: On selecting any of the extension under “Popular” and “Recommended” it’s state is not announced to the users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode#77062]
**Environment Details:**
VSCode Version : 1.35.6 (user setup)
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 4.1.2
ScreenReader Versions : NVDA2019.1.1, JAWS 2019

**Repro Steps:**
1.	Open Visual studio code editor.
2.	Using "Ctrl+Shift+X" key navigate to the extension side view bar option.
3.	Using "Tab" key navigate to extension side bar options.
4.	Start screen reader.
5.	Navigate to the available extensions under “Popular” and “Recommended” and activate any one of them.
6.	Listen if its state is announced or not.

**Actual:**
When screen reader users navigate and activate the extensions available under “Popular” and “Recommended” the selected state is not announced to the screen reader users.

**Expected:**
When user selects any of the extension the selection state should be announced to the screen reader users.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the selection state of the interactive elements is not announced to the screen reader users then they will not know which element they have selected.

**MAS Reference:**
MAS 1.3.1 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}) 

**Attachment for Reference:**
[VSCode_SidebarBarOptions_AccessingTheExtensionbaroptions_ScreenReader Selection state of the extension is not announced.pptx](https://github.com/microsoft/vscode/files/3376372/VSCode_SidebarBarOptions_AccessingTheExtensionbaroptions_ScreenReader.Selection.state.of.the.extension.is.not.announced.pptx)

**Comment by isidorn** [comment]
It is announced once you nagivate through the list.
Closing as designed / invalid
